utils/recog_wav.sh

```diff
@@ -50,25 +55,75 @@ set -e
 set -u
 set -o pipefail
 
-# existence check
+case "${recipe_with_ver}" in
+    "tedlium.demo")
+        share_url="https://drive.google.com/open?id=1UqIY6WJMZ4sxNxSugUqp3mrGb3j6h7xe"
```

Comment:
we should mention how to add pretrained models like this to the `/CONTRIBUTING.md`.

Reply:
And update [wiki](https://github.com/espnet/espnet/wiki/Model-Zoo) how to use this `utils/recog_wav.sh`

Reply:
add `doc/recog_wav.md` or update `doc/tutorial.md` for non-developer usage